tests/lib/rules/forbid-component-props.js

```diff
@@ -81,6 +81,17 @@ ruleTester.run('forbid-component-props', rule, {
     ].join('\n'),
     options: [{forbid: ['style', 'foo']}],
     parserOptions: parserOptions
+  }, {
+    code: [
+      'var First = React.createClass({',
```

Comment:
ah, Thanks. I know 'stateless functional component' , just have no idea what does SFC meaning. Thanks for point out